terraform/deployments/govuk-publishing-platform/app_static.tf

```
@@ -23,6 +23,7 @@ module "static" {
   source = "../../modules/app"
 
   service_name                     = "static"
+  backend_virtual_service_names    = [] # TODO what services does Static use?
```

I don't think it uses any, so you could probably remove this TODO too.

```suggestion
  backend_virtual_service_names    = [] # Static doesn't use any other services
```